cmd/helm/list.go

```diff
@@ -126,6 +126,7 @@ func newListCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	f.IntVarP(&client.Limit, "max", "m", 256, "maximum number of releases to fetch")
 	f.IntVar(&client.Offset, "offset", 0, "next release name in the list, used to offset from start value")
 	f.StringVarP(&client.Filter, "filter", "f", "", "a regular expression (Perl compatible). Any releases that match the expression will be included in the results")
+	f.StringVarP(&client.Selector, "selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
```

Comment:
I would change this to `label` to be more like `kubectl`
i.e. selector --> label

Reply:
Might want to mention that it is only supported for ConfigMap and Secrets backend storage

Reply:
Added mentions

Reply:
Makes sense to use `selector`. I was looking at setting not the getting part.

Reply:
So we can just close this conversation or you are waiting some changes in this thread?